content/800-data-platform/050-data-proxy/900-considerations-limitations.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Considerations and limitations'
+metaTitle: 'Considerations'
+metaDescription: 'When you use Data Proxy for applications in serverless and edge runtimes, you should bear in mind a few considerations and learn about the current limitations.'
+tocDepth: 3
+---
+
+## Considerations
+
+### Connection to databases hosted with a provider different from AWS
+
+It is possible to connect to a database hosted on any provider as long as it is accessible from a public internet address. For best results and response times, you should configure Data Proxy in a region that is the same or as close as possible to the database.
+
+We are looking to expand the number of providers and regions in the future. If a region or provider is not yet supported, you can [submit a request](https://pris.ly/proxy-new-region). This will help us prioritize providers and regions to add next.
+
+### Recommended concurrent connections
+
+We recommend that you use a database that supports at least 20 concurrent connections.
+Setting a concurrency limit on AWS Lambda should not be required, but you should be aware that, if necessary, requests will be queued until they can be processed.
```

Comment:
```suggestion
You will probably not need to set a concurrency limit on AWS Lambda. However, requests are queued if necessary until they can be processed.
```

To avoid "should" and some passive voice.
Also, does this need to be on a separate line from line 18?